examples/nlp/gpt/train_gpt_dpo.py

```diff
@@ -59,10 +60,16 @@ def main(cfg) -> None:
         load_base_model_only=False,
         restore_path=cfg.pretrained_checkpoint.restore_from_path,
     )
-    ref_policy_state_dict = retrieve_model_state_dict_in_cpu(
-        ptl_model, megatron_amp_O2=cfg.model.get("megatron_amp_O2", False)
-    )
-    ptl_model.ref_policy_state_dict = ref_policy_state_dict
+
+    init_peft(ptl_model, cfg.model)
+
+    if cfg.model.peft.peft_scheme == "lora":
+        ptl_model.ref_policy_state_dict = None
```

Comment:
ref_policy_state_dict already defaults to none, maybe we should just do

```python
if cfg.model.peft.peft_scheme == "none":
    ref_policy_state_dict = retrieve_model_state_dict_in_cpu(
            ptl_model, megatron_amp_O2=cfg.model.get("megatron_amp_O2", False)
        )
        ptl_model.ref_policy_state_dict = ref_policy_state_dict
```

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/142/commits/4237f5eb311fc097ce5b032e3cba5df7201c70cd